eng/common/scripts/common.ps1

```diff
@@ -29,6 +29,18 @@ if (-not $LanguageShort)
   $LangaugeShort = $Language
 }
 
+if (-not $LanguageDisplayName)
+{
+  $LanguageDisplayName = $Language
+}
+
+# Given the metadata url under https://github.com/Azure/azure-sdk/tree/master/_data/releases/latest, 
+# the function will return the csv metadata back as part of response.
+function Get-CSVMetadata ([string]$MetadataUri) {
```

Comment:
Should this function go into another script like artifact-metadata-parsing or package-properties? We might also want to setting the default parameter so we could simply call it in the context of our language without passing a parameter.